wd_core.au3

```diff
@@ -1222,7 +1222,7 @@ Func _WD_Startup()
 	$iPID = ProcessExists($sFile)
 
 	If $_WD_DRIVER_DETECT And $iPID Then
-		__WD_ConsoleWrite($sFuncName & ": Existing instance of " & $sFile & " ( PID=" & $iPID & " ) detected!")
+		 $sExistingDriver = "Existing instance of " & $sFile & " detected! (PID=" & $iPID & ")"
```

Comment:
After this modification it will never be used.

Reply:
I'm not sure what you mean by that. 🤔 This was the results from my testing --

### No prior instance

```
_WD_Startup: OS:	WIN_10 WIN32_NT 22000 
_WD_Startup: AutoIt:	3.3.14.5
_WD_Startup: Webdriver UDF:	0.9.0 (Update available)
_WD_Startup: WinHTTP:	1.6.4.2
_WD_Startup: Driver:	chromedriver.exe (32 Bit)
_WD_Startup: Params:	--verbose --log-path="c:\Users\danpo\Dropbox\webdriver\chrome.log"
_WD_Startup: Port:	9515
_WD_Startup: Command:	"chromedriver.exe" --verbose --log-path="c:\Users\danpo\Dropbox\webdriver\chrome.log" 
```

### Prior instance

```
_WD_Startup: OS:	WIN_10 WIN32_NT 22000 
_WD_Startup: AutoIt:	3.3.14.5
_WD_Startup: Webdriver UDF:	0.9.0 (Update available)
_WD_Startup: WinHTTP:	1.6.4.2
_WD_Startup: Driver:	chromedriver.exe (32 Bit)
_WD_Startup: Params:	--verbose --log-path="c:\Users\danpo\Dropbox\webdriver\chrome.log"
_WD_Startup: Port:	9515
_WD_Startup: Command:	Existing instance of chromedriver.exe detected! (PID=24764)
```

Reply:
I was just reading code not testing nor expanding the collapsed code.
my fault .... on GitHub code was collapsed.
![image](https://user-images.githubusercontent.com/11089482/172671275-3b59f3d9-f1ae-4856-982b-cfbcfd2de733.png)

This part should be treated as resolved.